index.js

```diff
@@ -239,6 +239,9 @@
   //  always2 :: a -> (b, c) -> a
   function always2(x) { return function(y, z) { return x; }; }
 
+  //  captureStackTrace :: Assignable a => (a, Function) -> a
+  var captureStackTrace = Error.captureStackTrace || noop;
```

Comment:
Should the return type be `Undefined` rather than `a`? As written `noop` should not be used here because it does not return its first argument.

Reply:
Oh, you're right. captureStackTrace does actually return `undefined`.